mockdata/appOwners.json

```diff
@@ -0,0 +1,63 @@
+[
+    
+    {
+        "id": "JsVz8OV6zqQnfklWU4Gz",
+        "username": "ankush",
+        "first_name": "test",
+        "last_name": "test",
+        "email": "test",
+        "phone": "test",
+        "yoe": "test",
+        "company": "test",
+        "designation": "test",
+        "img": "test",
+        "github_id": "AnkushDharkar",
+        "linkedin_id": "test",
+        "twitter_id": "test",
+        "instagram_id": "test",
+        "site": "test",
+        "isMember": "boolean",
+        "github_display_name": "Ankush Dharkar",
+        "incompleteUserDetails": false
+    },
+    {
+        "id": "JsVz8OV6zqQnfklWU4Gz",
+        "username": "sumit",
+        "first_name": "test",
+        "last_name": "test",
+        "email": "test",
+        "phone": "test",
+        "yoe": "test",
+        "company": "test",
+        "designation": "test",
+        "img": "test",
+        "github_id": "sumitd94",
+        "linkedin_id": "test",
+        "twitter_id": "test",
+        "instagram_id": "test",
+        "site": "test",
+        "isMember": "boolean",
+        "github_display_name": "Sumit Dhanania",
+        "incompleteUserDetails": false
+    },
+    {
+        "id": "JsVz8OV6zqQnfklWU4Gz",
+        "username": "nikhil",
+        "first_name": "Nikhil",
+        "last_name": "Bhandarkar",
+        "email": "test",
+        "phone": "test",
+        "yoe": "test",
+        "company": "test",
+        "designation": "test",
+        "img": "test",
+        "github_id": "whydonti",
+        "linkedin_id": "test",
+        "twitter_id": "test",
+        "instagram_id": "test",
+        "site": "test",
+        "isMember": "boolean",
+        "github_display_name": "Nikhil Bhandarkar",
+        "incompleteUserDetails": false
+    }
```

Comment:
Why these many fields when you are using only `username` ? Let's keep this data simple by including `first_name, last_name, username`
Rest all the fields are not required.
Also please add other App owners
`kratika, ankur, swaraj` (these are their usernames)